WasatchNET/Spectrometer.cs

```diff
@@ -2392,6 +2413,13 @@ public virtual float laserPowerSetpointMW
             }
             set
             {
+                if (!eeprom.hasLaserPowerCalibration())
+                {
+                    logger.error("can't set laser power in mW without a laser power calibration");
+                    laserPowerSetpointMW_ = 0;
+                    return;
+                }
+

```

Comment:
This seemingly contradicts the comments above, since we actually won't let the value be set unless the EEPROM has the calibration.